tests/api_tests/test_ancillary_studies_cloud_tasks.py

```diff
@@ -105,3 +119,117 @@ def tearDown(self):
         self.clear_table_after_test("nph.enrollment_event_type")
         self.clear_table_after_test("nph.participant_event_activity")
         self.clear_table_after_test("nph.enrollment_event")
+
+
+class InsertNphIncidentTaskApiCloudTaskTest(BaseTestCase):
+
+    DATETIME_FORMAT = "%Y-%m-%d %H:%M:%S"
+    TIME = datetime.strptime(datetime.now().strftime(DATETIME_FORMAT), DATETIME_FORMAT)
+
+    def setUp(self) -> None:
+        super().setUp()
+        self.nph_participant_dao = NphParticipantDao()
+        self.nph_activity_dao = NphActivityDao()
+        self.nph_participant_event_activity_dao = NphParticipantEventActivityDao()
+        self.nph_incident_dao = NphIncidentDao()
+        self.faker = Faker()
+
+    def _create_nph_participant(self, participant_obj: Dict[str, Any]) -> Participant:
+        nph_participant = Participant(**participant_obj)
+        with FakeClock(self.TIME):
+            return self.nph_participant_dao.insert(nph_participant)
+
+    def _create_nph_activity(self, activity_obj: Dict[str, Any]) -> Activity:
+        nph_activity = Activity(**activity_obj)
+        with FakeClock(self.TIME):
+            return self.nph_activity_dao.insert(nph_activity)
+
+    def _create_nph_participant_event_activity(self, obj: Dict[str, Any]) -> ParticipantEventActivity:
+        nph_participant_event_activity = ParticipantEventActivity(**obj)
+        with FakeClock(self.TIME):
+            return self.nph_participant_event_activity_dao.insert(nph_participant_event_activity)
```

Comment:
No need for these, you can lean into `NphDataGenerator` for all the db inserts for initial data.